qupulse/hardware/awgs/tabor.py

```diff
@@ -142,6 +142,8 @@ class TaborSequencing(Enum):
     ADVANCED = 2
 
 
+volatile_mapping_entry = namedtuple('Mapping_Entry', 'aSeq adv_position position change_table')
```

Comment:
Is it good style to use the namedtuple like this? Also are the names unterstandable?

Reply:
I would
1. keep class names CamelCase
2. The type name and the first argument to namedtuple should be the same
3. Better use typing.NamedTuple to annotate types
4. Keep member names snake_case

```python
from typing import NamedTuple

MappingEntry = NamedTuple('MappingEntry', [('a_seq', int), ('adv_position', int), ...]
```